pkg/util/liveconfig/checkaccess.go

```diff
@@ -0,0 +1,18 @@
+package liveconfig
```

Comment:
Please document what checkAccess is. There is no explanation of what this option does and what are the implications of using it.

Reply:
Hi @petrkotas, definitely.  I have some info in the ADO wiki [here](https://msazure.visualstudio.com/AzureRedHatOpenShift/_wiki/wikis/AzureRedHatOpenShift.wiki/409184/CheckAccess-API-migration).  Let me know if this is ok or if there are any points missing.

I should also be in a habit of including more info on the jira child tickets.  There are some background info on the parent one in ARO-1800.  Let me know if you have any suggestion on how I can improve it.